pull_request_template.md

```diff
@@ -2,30 +2,16 @@
 
 _insert reason_
 
-Ticket: _link/ref to ticket/jira/cherwell/trello_
-
-Akamai ticket: _Cherwell ticket #_
+Related links: _link/ref to ticket/jira/cherwell/trello_
 
 ## Description of Changes
 
 _insert description_
 
-This is a:
-
-- [ ] Bug fix (non-breaking change which fixes an issue)
-- [ ] Refactor (non-breaking change)
-- [ ] New or improved tests (non-breaking change)
-- [ ] New feature (non-breaking change which adds functionality or implements non-functional requirement)
-- [ ] Breaking change (fix or feature that would cause existing functionality to change)
-- [ ] Documentation or design proposal
-
-
 ## Checklist
 
-- [ ] This PR has a descriptive title
-- [ ] My code follows the code style of this project.
 - [ ] My change requires a change to the documentation and docs have been updated.
 - [ ] I have added tests to cover my changes.
-- [ ] All new and existing tests passed.
-- [ ] This change requires an Akamai change.  Cherwell ticket number is in Reason for Change.
-- [ ] This needs a change request.  CR# is in title.
+- [ ] This change can be rolled back or has been marked as breaking.
```

Comment:
Is there better wording for the first two items in this list so they can be checked when they are not relevant?

Reply:
@vsudhire Perhaps a variation of your suggestion, since the change may require a doc update that's been completed.